common/config/azure-pipelines/jobs/version-bump.yaml

```diff
@@ -238,6 +238,9 @@ jobs:
           git commit -m "$(getVersion.version)" --author="imodeljs-admin <imodeljs-admin@users.noreply.github.com>"
```

Comment:
Another random line comment...

I think the npm publish step (in `templates/publish.yaml`, which is called from `jobs/fast-ci.yaml`) would run on the wrong commit, the one before you push eslint changes. Maybe you can change the check to look up the git tag or something else instead of looking at the commit message.